sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Get-SPOSiteManageVersionPolicyJobProgress.md

```diff
@@ -0,0 +1,70 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
```

Comment:
(applies to this file and the others)

- add the value for "online version"
- also add:
  - title
  - author <-- your GitHub alias
  - ms.author <-- your MSFT alias

take a look at other files for examples